python/pyspark/sql/tests/test_utils.py

```diff
@@ -648,17 +649,25 @@ def test_assert_error_pandas_df(self):
 
         self.check_error(
             exception=pe.exception,
-            error_class="UNSUPPORTED_DATA_TYPE",
-            message_parameters={"data_type": pd.DataFrame},
+            error_class="INVALID_TYPE_DF_EQUALITY_ARG",
+            message_parameters={
+                "expected_type": DataFrame,
+                "arg_name": "df",
+                "actual_type": pd.DataFrame,
```

Comment:
Just curious, what will the error message look like in this case? Is it clear that the method expects a pyspark dataframe instead of a pandas dataframe?